### PR TITLE
A script to generate multiapi readme files. OAV update.

### PR DESCRIPTION
**Pull request by sergey-shandar**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,16 +13,21 @@
...
     "mocha": "*",
     "oad": "^0.1.11",
-    "oav": "^0.11.1",
+    "oav": "^0.13.2",
     "request": "^2.61.0",
     "request-promise-native": "^1.0.5",
-    "z-schema": "^3.24.2"
+    "typescript": "^3.2.4",
+    "z-schema": "^3.25.0",
+    "ts-node": "^8.0.1"
```
Comment (nschonni) [inline package.json]: Think this needs to move up, or the next install will re-order it automatically

**Comment by sergey-shandar** [comment]
See also generated readme files https://github.com/Azure/azure-rest-api-specs/pull/5080/files

**Review by veronicagg — CHANGES_REQUESTED** [review]
Few questions before approving, regarding the output, the readmes just contain the files to pick up for generation and no other settings, so they won't be suitable to replace any of the existing. If they are used for code generation, how are we consuming the other setting from existing readme files?
Also I see that we're picking up all the file paths and putting them together for generation, is that what the muti-api feature will take as input? 
Code itself looks good to me.

**Comment by sergey-shandar** [comment]
@veronicagg, @fearthecowboy knows the answers but, I suspect, it's on early stages of development so we cane address these issues after.

**Review by veronicagg — APPROVED** [review]

**Pull request merged**